readme.md

```diff
@@ -163,48 +169,53 @@ How are we defining columns in `create_table`?
 * We then use methods like `.string` or `.integer` to represent the data types of the columns we are creating.
 * These methods take in a symbol argument (e.g., `:photo_url`) that represents the name of the column.  
 
```

Comment:
Referring to line 153 ("Now we need to define... ), don't see a need for a bullet point there.